test_sets/PTU_no_certificate_exists_at_module_config.txt

```diff
@@ -0,0 +1,57 @@
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/045_ATF_P_TC_navi_PTU_brings_certificate_Audio_ForceProtectedService_ON.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/046_ATF_P_TC_navi_PTU_brings_certificate_Video_ForceProtectedService_ON.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/047_ATF_P_TC_navi_PTU_brings_certificate_RPC_ForceProtectedService_ON.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/048_ATF_P_TC_navi_PTU_brings_certificate_Hybrid_ForceProtectedService_ON.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/051_ATF_P_TC_media_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/052_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_ON.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/053_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_ON.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/054_ATF_P_TC_media_PTU_brings_certificate_RPC_ForceProtectedService_ON.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/055_ATF_N_TC_media_app_not_first_RPC_encrypt_true.lua Check_that_SDL_will_not_trigger_PTU_when_second_RPC_service_sent_encrypted_true
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/056_ATF_N_TC_comm_app_not_first_RPC_encrypt_true.lua Check_that_SDL_will_not_trigger_PTU_when_second_RPC_service_sent_encrypted_true
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/057_ATF_N_TC_default_app_not_first_RPC_encrypt_true.lua Check_that_SDL_will_not_trigger_PTU_when_second_RPC_service_sent_encrypted_true
```

Comment:
@istoimenova  Please clarify: 055-057
First service started secure service or not in precondition for this case?